src/index.html

```diff
@@ -3,9 +3,33 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wbpack Exercise</title>
+    <title>Exercise</title>
 </head>
 <body>
-    <h1>Hello webpack!</h1>
+    <main>
+        <header>
+            <h1>Leaderboard</h1>
+        </header>
+        <section>
+            <div id="scores">
+                <div id="column1">
+                    <div id="heading">
+                        <h2>Recent scores</h2>
+                        <button class="btnstyle" id="refreshbtn">Refresh</button>
+                    </div>
+                    <div id="scorediv">
+                    </div>
+                </div>
+                <div id="column2">
+                    <h2>Add your score</h2>
+                    <form>
+                        <input type="text" id="name" name="fname" placeholder="your name" required ><br><br>
+                        <input type="text" id="score" name="lname" placeholder="your score" required ><br><br>
+                        <input type="submit" value="Submit" id="submitbtn">
+                    </form> 
+                </div>
+            </div>
+        </section>
+    </main>
 </body>
 </html>
```

Comment:
### README FILE

- [ ] Please, let's improve the readme file by updating the following to make the readme more professional below:



:point_down: *Let's update the link to clone the project to this current project*
![Screenshot from 2023-02-04 13-57-27](https://user-images.githubusercontent.com/59397159/216769038-e58e3290-8003-4a99-8a64-1c89fb648409.png)



:point_down: *Kindly remove the live demo link since you did not deploy the app as the link you added is a broken link*
![Screenshot from 2023-02-04 13-56-51](https://user-images.githubusercontent.com/59397159/216769039-75e5f44f-f083-4208-809e-a833880cf411.png)

:point_down: *This should be an Array of Scores and not book because the project is mainly a score board. Update this Please.*
![Screenshot from 2023-02-04 13-56-23](https://user-images.githubusercontent.com/59397159/216769042-6fc3e375-5dff-47f5-acb7-1f3466754468.png)

:point_down: *Let's update the name to the name of this current project*
![Screenshot from 2023-02-04 13-55-51](https://user-images.githubusercontent.com/59397159/216769044-e57c8089-e376-4649-9897-3a4865a1603d.png)

:point_down: *Let's add a license as the link is currently broken*
![Screenshot from 2023-02-04 14-07-25](https://user-images.githubusercontent.com/59397159/216769444-ac0c1925-596c-45c1-b381-9dace291586b.png)